src/ripple/nodestore/ShardSizeTuning.md

```diff
@@ -141,6 +141,32 @@ sizes of nudb.* which are permanent for deterministic shards.
 We can see that in all tables ratio is greater then 1, so using shards with
 16 ledgers is preferred.
 
+## Compare 16K shards and 32K shards
+
+To claim that shards with 16K ledgers are the best choise, we also assembled
+shards with 32 ledgers per shard. There were 11 such a shards computed with
+indexes from 1300 to 1800 with step 50. These are the shards which correspond
+to previously calculated shards with 16 ledgers.
+
+Here are results for these shards. Average size of nudb.dat is 6.74G, average
+sum of sizes for nudb.dat and nudb.key is 7.65G. Now we can compare these sizes
+and doubled average sizes for 16k-ledgers shards. We have
+
+3.89 * 2 = 7.78 greather than 6.74 by 15 percents.
+
+4.43 * 2 = 8.86 greather than 7.65 by 16 percents.
+
 ## Conclusion
 
-We recommend use shards with 16K ledgers.
+We showed that using shards with 8k ledgers leads to raising required disk
+size by 20% in comparison with using shards with 16k ledgers. In the same way,
+using shards with 16k ledgers leads to raising required disk space by 15%
+in comparison with using shards with 16k ledgers. So if we only take into
+consideration disk usage, then optimal is using shards with 32k ledgers.
```

Comment:
...if that's your only option on the table. I bet that if you try 64k shards or 128k shards, there will be additional savings. ;-)

Reply:
Added another argument.

Reply:
Saving ~10% of storage space on a several TiB data set is not worth the effort according to this argument? I'm not totally convinced by that one to be honest.

Reply:
This is secondary reason, the main is problems with acquiring large shards.